src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier.php

```diff
@@ -57,6 +59,10 @@
             ->args([service('mailer.transports'), service('messenger.default_bus')->ignoreOnInvalid()])
             ->tag('notifier.channel', ['channel' => 'email'])
 
+        ->set('notifier.channel.web_push', WebPushChannel::class)
+            ->args([service('texter.transports'), service('messenger.default_bus')->ignoreOnInvalid()])
+            ->tag('notifier.channel', ['channel' => 'web_push'])
```

Comment:
You decided to use `web_push` to keep `push` for something like iOS and/or Android push notifications?

Reply:
Not sure really, my target just was that I could send notifications to those who wants web push notifications.. But I think most providers allows also and mobile push so that's debatable

Reply:
In this case I would be in favour of using `push` and a `PushMessage` instead of having a `web_`/`Web` prefix everywhere.

@fabpot can you please leave your 2cents here?